python/pycryptodome/security/insecure-cipher-algorithm-blowfish.py

```diff
@@ -25,10 +25,12 @@
 bs = pycrypto_blowfish.block_size
 # ruleid:insecure-cipher-algorithm-blowfish
 cipher = pycrypto_blowfish.new(key, pycrypto_blowfish.MODE_CBC, iv)
+# deepruleid:insecure-cipher-algorithm-blowfish
 msg = iv + cipher.encrypt(plaintext + padding)
 bs = pycryptodomex_blowfish.block_size
 # ruleid:insecure-cipher-algorithm-blowfish
 cipher = pycryptodomex_blowfish.new(key, pycryptodomex_blowfish.MODE_CBC, iv)
+# deepruleid:insecure-cipher-algorithm-blowfish
 msg = iv + cipher.encrypt(plaintext + padding)
 
```

Comment:
NOTE: all of these pycryptodome findings were introduced as result of uncommenting the line that wipes id_info in generic v generic; I am not 100% sure if these findings are expected. Will need to check if this is indeed the right behavior.